include/gatekeeper_gt.h

```diff
@@ -226,6 +227,11 @@ int run_gt(struct net_config *net_conf, struct gt_config *gt_conf,
 int l_update_gt_lua_states(lua_State *l);
 int l_update_gt_lua_states_incrementally(lua_State *l);
 
+uint16_t gt_cpu_to_be_16(uint16_t x);
+uint32_t gt_cpu_to_be_32(uint32_t x);
+uint16_t gt_be_to_cpu_16(uint16_t x);
+uint32_t gt_be_to_cpu_32(uint32_t x);
```

Comment:
Drop all these declarations because these four functions are only needed in Lua policies.

Reply:
If we drop these declarations, then there is a compilation error: ``no previous prototype for ‘gt_cpu_to_be_16’ [-Werror=missing-prototypes]``

Reply:
In this case, move these prototypes to file `gt/main.c` before the definition of the functions and add a comment to explain why they are needed.